gotpm/cmd/seal_test.go

```diff
@@ -104,16 +129,27 @@ func TestUnsealFail(t *testing.T) {
 			sealedFile := makeTempFile(t, nil)
 			defer os.Remove(sealedFile)
 
-			RootCmd.SetArgs([]string{"seal", "--quiet", "--input", secretFile, "--output", sealedFile, "--pcrs", "23"})
+			sealArgs := []string{"seal", "--quiet", "--input", secretFile, "--output", sealedFile}
+			if test.sealPCRs != "" {
+				sealArgs = append(sealArgs, "--pcrs", test.sealPCRs)
+			}
+			RootCmd.SetArgs(sealArgs)
 			if err := RootCmd.Execute(); err != nil {
 				t.Error(err)
 			}
+			initPCRs()
```

Comment:
just manually set pcrs here (and add a note explaining why)

Reply:
done